js/main.js

```diff
@@ -25,9 +25,7 @@ function logForm(event) {
   $setImg.setAttribute('src', 'images/placeholder-image-square.jpg');
   $formSubmit.reset();
   $ul.prepend(entryDOM(formObject));
-  $newEntry.className = 'hidden';
-  $entriesPage.className = '';
-  data.view = 'entries';
+  switchViews($entriesPage.getAttribute('data-view'));
```

Comment:
You can just change this to `switchView("entries")` since you know you want to go to that view.